samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/.gitignore

```diff
@@ -0,0 +1,405 @@
+
```

Comment:
Could you please update this .gitignore to use .dll?

Btw, these were created via gitignore.io website :)